models/base/manifest/<YOUR PACKAGE NAME>_base_sessions_lifecycle_manifest.sql

```diff
@@ -0,0 +1,49 @@
+{#
+Copyright (c) 2023-present Snowplow Analytics Ltd. All rights reserved.
+This program is licensed to you under the Snowplow Personal and Academic License Version 1.0,
+and you may not use this file except in compliance with the Snowplow Personal and Academic License Version 1.0.
+You may obtain a copy of the Snowplow Personal and Academic License Version 1.0 at https://docs.snowplow.io/personal-and-academic-license-1.0/
+#}
+
+{{
+  config(
+    materialized='incremental',
+    unique_key='session_identifier',
+    upsert_date_key='start_tstamp',
+    sort='start_tstamp',
+    dist='session_identifier',
+    partition_by = snowplow_utils.get_value_by_target_type(bigquery_val={
+      "field": "start_tstamp",
+      "data_type": "timestamp"
+    }, databricks_val='start_tstamp_date'),
+    cluster_by=snowplow_utils.get_value_by_target_type(bigquery_val=["session_identifier"], snowflake_val=["to_date(start_tstamp)"]),
```

Comment:
Can we swap this out for a macro instead please, as a customer wanted that in unified